projects/admin/src/app/record/custom-editor/user-id-editor/user-id-editor.component.ts

```diff
@@ -82,15 +85,13 @@ export class UserIdEditorComponent implements OnInit {
       tap(
         schema => {
           if (schema != null) {
+            schema = formToWidget(schema.schema, this._loggerService);
             this.fields = [
-              this._formlyJsonschema.toFieldConfig(orderedJsonSchema(schema.schema), {
+              this._formlyJsonschema.toFieldConfig(orderedJsonSchema(schema), {
```

Comment:
Do we need to call `formToWidget` as `initialRequired` has been added?

Reply:
This is the case 2 lines above